src/modules/local_position_estimator/sensors/flow.cpp

```diff
@@ -172,10 +172,10 @@ void BlockLocalPositionEstimator::flowCorrect()
 	Matrix<float, n_y_flow, n_y_flow> S = C * _P * C.transpose() + R;
 
 	// publish innovations
-	_pub_innov.get().flow_innov[0] = r(0);
-	_pub_innov.get().flow_innov[1] = r(1);
-	_pub_innov.get().flow_innov_var[0] = S(0, 0);
-	_pub_innov.get().flow_innov_var[1] = S(1, 1);
+	_pub_innov.get().flow[0] = r(0);
+	_pub_innov_var.get().flow[1] = r(1);
+	_pub_innov.get().flow[0] = S(0, 0);
+	_pub_innov_var.get().flow[1] = S(1, 1);
```

Comment:
This is incorrect, please check.

Reply:
Thank you very much for catching that. I really appreciate your detailed reviews.